README.md

```diff
@@ -14,6 +14,7 @@ You will need the following things:
 creating one can be found [here](https://developer.pingidentity.com/content/p14c/en/signup.html). Also make sure that it is enabled plus redirect URL's and 
 access grants by scopes are properly set.
 - At least one user in the same environment as the application (not assigned)
+- Installed [Node.js](https://nodejs.org/en/download/)
```

Comment:
Nit: I think it would read better as "To have installed [Node.js]". Then the full sentence is "You will need to have installed..."

Reply:
I've added only a shortcut `Installed` , because it belong to the top [part above](https://github.com/pingidentity/pingone-customers-sample-oidc/pull/4/files#diff-04c6e90faac2675aa89e2176d2eec7d8R10): `You will need the following things:`.

Reply:
Right, which reads, "You will need the following things: Installed..."

"You will need the following things: To have installed..." sounds better to me. I won't split hairs though, if you don't want to change it I'll still approve

Reply:
Updated as was proposed